exercises/simple-linked-list/tests/simple-linked-list.rs

```diff
@@ -0,0 +1,63 @@
+extern crate simple_linked_list;
+
+use simple_linked_list::SimpleLinkedList;
+
+#[test]
+fn test_new_list_is_empty() {
+    let list: SimpleLinkedList<u32> = SimpleLinkedList::new();
+    assert_eq!(list.size(), 0, "list's size must be 0");
+}
+
+#[test]
+#[ignore]
+fn push_increments_size() {
+    let mut list: SimpleLinkedList<u32> = SimpleLinkedList::new();
+    list.push(1);
+    assert_eq!(list.size(), 1, "list's size must be 1");
+    list.push(2);
+    assert_eq!(list.size(), 2, "list's size must be 2");
+}
+
+#[test]
+#[ignore]
+fn pop_decrements_size() {
+    let mut list: SimpleLinkedList<u32> = SimpleLinkedList::new();
+    list.push(1);
+    list.push(2);
+    list.pop();
+    assert_eq!(list.size(), 1, "list's size must be 1");
+    list.pop();
+    assert_eq!(list.size(), 0, "list's size must be 0");
+}
+
+#[test]
+#[ignore]
+fn pop_returns_last_added_element() {
+    let mut list: SimpleLinkedList<u32> = SimpleLinkedList::new();
+    list.push(1);
+    list.push(2);
+    assert_eq!(list.pop(), Some(2), "Element must be 2");
+    assert_eq!(list.pop(), Some(1), "Element must be 1");
+    assert_eq!(list.pop(), None, "No element should be contained in list");
+}
+
+#[test]
+#[ignore]
+fn peek_returns_current_element() {
```

Comment:
Nitpick: `peek_returns_head_element` would be more descriptive than `peek_returns_current_element`. If you feel strongly otherwise, please feel free to ignore this comment.